atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -483,14 +485,21 @@ private TransactionManager serializableInternal(@Output List<AutoCloseable> clos
                 getSchemaMetadataCoordinationService(metricsManager, lockAndTimestampServices, internalKeyValueService);
 
         SettableFuture<MultiTableSweepQueueWriter> initialisableWriter = SettableFuture.create();
+
+        PuncherStore keyValuePuncherStore =
+                KeyValueServicePuncherStore.create(internalKeyValueService, config().initializeAsync());
+        PuncherStore cachingPuncherStore =
+                CachingPuncherStore.create(keyValuePuncherStore, config().getPunchIntervalMillis() * 3);
```

Comment:
not actionable: this looks incredibly random, but matches existing behaviour in DCB. To be fair three times the interval doesn't seem super unreasonable